build/helm/keylime/charts/keylime-registrar/values.yaml

```diff
@@ -20,6 +20,9 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+roleName: "registrar-role"
+roleBindingName: "registrar-rolebinding"
```

Comment:
@sarroutbi sorry, just seeing this now... can these not be derived from release name / release information?

Reply:
basically, similarly to how it's done for service accounts for example if you understand what I mean

Reply:
Yes, I think I understand. I will update the PR to include a template similar to the one used in service accounts